whats_fresh/whats_fresh_api/templates/story.html

```diff
@@ -151,14 +277,14 @@ <h4 id='error_tag' class='light warning'></h4>
     function deleteVideo(element)
     {
         $( element ).parent().css("display", "none");
-        if ($( ".video:visible" ).length === 0) {
-            $( ".no_products" ).show();
+        if ($( ".video:visible" ).length === 0 && $( ".video:visible" ).length === 0 ) {
```

Comment:
``if ($( ".video:visible" ).length === 0 && $( ".video:visible" ).length === 0 ) {``

the two terms on either side of the && are exactly the same here and for .image:visible.